ci/lint.yml

```diff
@@ -0,0 +1,13 @@
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: golang
+    tag: 1.15.7
```

Comment:
I think this should be `onsdigital/dp-concourse-tools-java` not golang

Reply:
ok, now changed to latest image for java